presidio-anonymizer/presidio_anonymizer/anonymizers/anonymizer.py

```diff
@@ -9,6 +9,6 @@ class Anonymizer(ABC):
     logger = logging.getLogger("presidio-anonymizer")
 
     @abstractmethod
-    def anonymize(self, params={}):
+    def anonymize(self, original_text=None, params={}):
```

Comment:
Please notice, not all anonymize methods require the original_text.

Reply:
The original_text can be passed through params
or better yet we can change params to **kwargs

Reply:
I know, but can we avoid it in the interface level?

Reply:
yes, use **kwargs instead of params.
params was a temporary place holder

Reply:
**kwargs essentially let you pass anything so it misses the idea of an interface.
Interface changes between implementations, so it breaks the open close principle